msteams-platform/concepts/deep-links.md

```diff
@@ -62,8 +64,32 @@ Examples:
 > var taskItemUrl = 'https://teams.microsoft.com/l/entity/fe4a8eba-2a31-4737-8e33-e5fae6fee194/tasklist123?webUrl=' + encodedWebUrl + '&context=' + encodedContext;
 > ```
 
-## Consuming a deep link from a tab
+### Consuming a deep link from a tab
 
 When navigating to a deep link, Microsoft Teams simply navigates to the tab and provides a mechanism via the Microsoft Teams JavaScript library to retrieve the sub-entity ID (if it exists).
 
 The [`microsoftTeams.getContext`](/javascript/api/msteams-client/microsoftteams.global) call returns a context that includes the `subEntityId` field if the tab was navigated to via a deep link.
+
+## Deep linking to a chat
+> [!NOTE]
+> This functionality is currently available in Developer Preview
+
+You can now create deep links to private chats between users. You can link to an existing chat, or you can specify a set of chat participants to start a new chat. Optionally, you can specify the name of the chat, along with text that should be inserted into the user's compose box. New chats will be created in draft state until the user sends the first message. You can think of this feature as a shortcut for a basic user action.
+
+As an example use case, if you are returning an Office 365 user profile from your bot as a card, this deep link can allow the user to easily chat with that person.
+
+### Generating a deep link to a chat
+
+Use this format for a deep link that you can use in a bot, Connector, or messaging extension card:
+
+`https://teams.microsoft.com/l/chat/0/0?users=<user1>,<user2>,...&topicName=<chat name>&message=<precanned text>`
+
+Example: `https://teams.microsoft.com/l/chat/0/0?users=joe@contoso.com,bob@contoso.com&topicName=Prep%20For%20Meeting%20Tomorrow&message=Hi%20folks%2C%20kicking%20off%20a%20chat%20about%20our%20meeting%20tomorrow`
+
+The query parameters are:
+
+* `users`&emsp;The comma-separated list of user IDs, which can be either AAD UserPrincipalName (typically an email address) or User ID value
```

Comment:
"or User ID value": let's be clear on what this user id is: AAD OID? 29:xxx id? (something else?)